src/neo/Persistence/SnapshotView.cs

```diff
@@ -0,0 +1,46 @@
+using Neo.IO.Caching;
+using Neo.IO.Wrappers;
+using Neo.Ledger;
+using System;
+
+namespace Neo.Persistence
+{
+    /// <summary>
+    /// Provide a <see cref="StoreView"/> for accessing snapshots.
```

Comment:
ReadOnly as well?
If so, maybe `ReadOnlySnapshotView`?

Reply:
This is not readonly. It is a replacement of the old class `Snapshot`.